perfmetrics/scripts/continuous_test/ml_tests/tf/setup_scripts/setup_container.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+# Install go lang
+wget -O go_tar.tar.gz https://go.dev/dl/go1.19.4.linux-amd64.tar.gz
+sudo rm -rf /usr/local/go && tar -xzf go_tar.tar.gz && sudo mv go /usr/local
+export PATH=$PATH:/usr/local/go/bin
+
+# Clone the repo and build gcsfuse
```

Comment:
Add comment that we need to change this code once log rotation feature is implemented.